### PR TITLE
Feature/lw 15 users confirmation

### PR DESCRIPTION
**Pull request by NDragneelL9**
This task is depented on LW-13/Recover-password, so i merged lw-13 into this branch.

### CODE REVIEW

**Review by kel-ln — APPROVED** [review]

**Review by MonkeyKingBar-bit — APPROVED** [review]

File: app/models/user.rb [hunk 1]
```diff
@@ -3,22 +3,28 @@
 class User < ApplicationRecord
   audited
 
+  before_validation :generate_password, on: :create
   # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  devise :database_authenticatable, :registerable,
+  # :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable, :confirmable,
```
Comment (Lonely-fr) [inline app/models/user.rb]: fix seeds or remove ':confirmable'

**Review by Lonely-fr — CHANGES_REQUESTED** [review]

File: config/environment.rb [hunk 2]
```diff
@@ -2,14 +2,4 @@
...
-  address:              'smtp.gmail.com',
-  port:                 587,
-  domain:               'gmail.com',
-  user_name:            ENV['MAILER_EMAIL_ADDRESS'],
-  password:             ENV['MAILER_EMAIL_PASSWORD'],
-  authentication:       :plain,
-  enable_starttls_auto: true
-}
\ No newline at end of file
+Rails.application.initialize!
```
Comment (Laranto-spb) [inline config/environment.rb]: Where did this config move to? Just for my understanding

**Review by Laranto-spb — APPROVED** [review]

File: config/environment.rb (hunk 2 above)
Comment (NDragneelL9) [reply config/environment.rb]: we already have it inside environments/production, and for test/development we use letter_opener.

File: app/models/user.rb (hunk 1 above)
Comment (NDragneelL9) [reply app/models/user.rb]: fixed

**Review by Lonely-fr — APPROVED** [review]

**Review by alllpim — APPROVED** [review]

**Review by helmy-er — APPROVED** [review]

**Pull request merged**